pkg/security/security_profile/profile/profile_dir.go

```diff
@@ -0,0 +1,384 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+// +build linux
+
+package profile
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"sort"
+	"sync"
+	"time"
+
+	"github.com/fsnotify/fsnotify"
+	"github.com/skydive-project/go-debouncer"
+	"golang.org/x/exp/slices"
+
+	proto "github.com/DataDog/datadog-agent/pkg/security/proto/security_profile/v1"
+	cgroupModel "github.com/DataDog/datadog-agent/pkg/security/resolvers/cgroup/model"
+	"github.com/DataDog/datadog-agent/pkg/security/seclog"
+	"github.com/DataDog/datadog-agent/pkg/security/utils"
+)
+
+var (
+	debounceDelay        = 5 * time.Second
```

Comment:
will be more clear if called workloadSelectorDebounceDelay ?

Reply:
👍🏻 on it !